script/if_filter_delete.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright 2023 Intel Corporation
+
+set -e
+
+if [ $# -lt 1 ]; then
+    echo "Please special the network interface"
```

Comment:
typo: special -> specify could be better

Reply:
3x